lib/JobProperties.pm

```diff
@@ -190,11 +186,6 @@ sub _set_defaults
 	$self->set_file('shuf', $shuf) if ((defined $shuf) and (-e $shuf));
 	
 	$self->set_file('nucmer', $nucmer) if ((defined $nucmer) and (-e $nucmer));
```

Comment:
Line removed from JobProperties.pm in commit  b9e39151deb744ee8bc3c0c41903bfe4334b4de.